sys/log/src/log_nmgr.c

```diff
@@ -180,7 +431,7 @@ log_nmgr_clear(struct nmgr_jbuf *njb)
         }
 
         rc = log_flush(log);
-        if (rc != 0) {
+        if (rc) {
```

Comment:
Yes, I agree but I prefer keeping it clean. Doesn't hurt and I am not violating the coding standards :-)